proposals/001-funding-timeout-recovery.md

```diff
@@ -0,0 +1,179 @@
+# Funding Timeout Recovery
+
+> "So long, and thanks for all the sigs" 
+>     -- Hitchhikers guide to Lightning
+
+
+## Problem description
+
+Due to circumstances it may take considerable time for a funding
+transaction to confirm, e.g., the funder chose to attach a fee that is
+too low to guarantee timely confirmation. This means the channel
+funding may not complete for quite some time, and the specification
+thus allows the fundee to forget the channel after 2016 blocks:
+
+>  - SHOULD forget the channel if it does not see the correct funding
+>    transaction after a timeout of 2016 blocks.
+
+Since this leaves the funder in a situation where they are forced
+to use the commitment transaction, and since the commitment
+transaction is using an overestimated feerate, this could end up
+costing the funder more than strictly necessary, and the specification
+suggests avoiding this situation thus:
+
+> If the fundee forgets the channel before it was confirmed, the
+> funder will need to broadcast the commitment transaction to get his
+> funds back and open a new channel. To avoid this, the funder should
+> ensure the funding transaction confirms in the next 2016 blocks.
+
+However, despite a funder's best efforts to estimate the funding
+transaction fee correctly, the estimation may still fail to guarantee
+timely confirmation.
```

Comment:
You could note that this isn't the end of the story: the funder could use CPFP to ensure the funding tx confirms in a timely manner. But he could choose *not to do it* if the on-chain fees are high and he actually wants to give up on this channel and wait for lower fees (or even with CPFP fees could keep rising and he misses the 2016 block time window).

Reply:
Well, the described effect is still the same, isn't it? The funding transaction doesn't confirm in time (best effort here stands for any shenanigans he might try, including RBF, CPFP, bribing miners, etc).